packages/framework/src/Model/Product/Elasticsearch/ElasticsearchExportCronModule.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Model\Product\Elasticsearch;
+
+use Shopsys\Plugin\Cron\SimpleCronModuleInterface;
+use Symfony\Bridge\Monolog\Logger;
+use Symfony\Component\Console\Output\ConsoleOutput;
+
+class ElasticsearchExportCronModule implements SimpleCronModuleInterface
+{
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Product\Elasticsearch\ElasticsearchExportProductFacade
+     */
+    private $elasticsearchExportProductFacade;
+
+    /**
+     * @var \Symfony\Component\Console\Output\ConsoleOutput
+     */
+    private $consoleOutput;
+
+    public function __construct(ElasticsearchExportProductFacade $elasticsearchExportProductFacade, ConsoleOutput $consoleOutput)
+    {
+        $this->elasticsearchExportProductFacade = $elasticsearchExportProductFacade;
+        $this->consoleOutput = $consoleOutput;
```

Comment:
This is because `ElasticsearchExportProductFacade` is designed to write into output. We can change this Cron Module to use `NullOutput`, but we will loose a bit of information.

In fact I have no idea if it is useful for cron module to write something into output, is it?

Reply:
Me too :D That's why I compared your module with others cron modules.